### ISSUE TITLE
Flaky dropout test in tfjs-layers

### ISSUE THREAD

**Issue by dsmilkov** [tensorflow/tfjs#1894]
Chrome 76.0.3809 (Mac OS X 10.13.6) Dropout Layer cpu undefined tensor training=true, dropoutRate=0.5, noiseShape=[2,3,1] FAILED
Expected 24 to be less than 24.

See logs: https://pantheon.corp.google.com/cloud-build/builds/1cd1f7aa-190b-44a7-8464-162106b4a737?project=learnjs-174218

Related to this PR: https://github.com/tensorflow/tfjs-layers/pull/556
